packages/upswyng-core/src/ResourceSchedule.ts

```diff
@@ -47,6 +52,51 @@ function validateTimezone(timezone: string): TTimezoneName {
   return timezone as TTimezoneName;
 }
 
+function makeSchedulePeriodDate(dt: Date, timeValue: string): Date {
```

Comment:
what's timevalue?

Reply:
Oh is that like 8:00 PM - 2:00 AM Next Day ?

Reply:
Time value comes is the `TScheduleItem.toTime.value` (or `fromTime`). It takes that to create the JS date equivalent of the `dt`, in this case the next occurence's date, and the time value.